### DISCUSSION TITLE
Location search

### DISCUSSION THREAD

**Discussion by turboscrew** [Stellarium/stellarium#2322] [Q&A] [answered]
It seems like the location search no longer works. Not on Windows 11 nor on Linux Mint.
Is there any other way to "select" the pole (RA 0h0m0s, DEC +90°0'0")?
I'd like to see, in the ocular view, which direction, from the pole, the polaris is now. It helps in polar alignment.

**Comment by gzotti** [comment, marked as answer]
We have had a few reports on problems with the coordinate input interface. A current beta should have this fixed. 
https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Reply by turboscrew** [reply]
I tried also Alt-Shift-N, and it took me to the pole, but when I tried to use ocular view, it told me to select target first.
But I'll have a look into that beta, thanks!

BTW, I wonder if there's some sense adding predefined targets for search such as celestial poles and First Point of Aries?

**Reply by gzotti** [reply]
You may want to disable the requirement to select an object first then, in the Oculars settings dialog.

**Reply by turboscrew** [reply]
YES! THANKS!
It was unselected, but started to work when I first selected and then unselected it.

**Reply by turboscrew** [reply]
I would have liked to mark the second answer by gzotti as an answer, but didn't know how.
